virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -126,6 +131,8 @@ Deprecated features are included in the current release and supported. However,
 //CNV-32032 Release note: DEPRECATED FEATURE (Windows 2012R2 templates deprecated)
 * Support for Windows Server 2012 R2 templates is deprecated.
 
+//CNV-34681: Deprecated alerts
+* The alerts `KubeVirtComponentExceedsRequestedMemory` and `KubeVirtComponentExceedsRequestedCPU` are deprecated. You can safely ignore or xref:../../observability/monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence] them.
```

Comment:
ignore the -> silence them

Reply:
Hi @sradco , do you mean I should change it to just say:
```suggestion
* The alerts `KubeVirtComponentExceedsRequestedMemory` and `KubeVirtComponentExceedsRequestedCPU` are deprecated. You can safely xref:../../observability/monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence] them.
```